woocommerce/woocommerce-android
# Display composite products

⚠️ This PR is missing the navigation to the component details that will be handled in a different PR.
⚠️ This PR depends on [this FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2714)

Closes: #8405

### Why
We are adding Composite Products read-only support to the Woo App.

### Description
This PR adds support to see the product's components information in the product details screen. To achieve this goal, we are getting the information from the new CompositeComponents fields and displaying the list of components accordingly.  

### Testing instructions

#### Prerequisites

1. Install and activate the [Composite Products](https://woocommerce.com/products/composite-products/) extension on your store.
2. Create at least one product with the Composite product type, with at least one component.

#### To test

1. Build and run the app.
2. Open the Products tab and select a composite product.
3. Tap the Components row in the product details. 
4. Confirm the app navigates to the Components list.


### Images/gif

https://user-images.githubusercontent.com/18119390/231668297-4a004dd4-3339-4fde-b176-79332a81ce0b.mp4
